app/Http/Controllers/Backend/UserController.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace App\Http\Controllers\Backend;
+
+use Illuminate\Http\Request;
+use App\Http\Requests;
+use App\Models\User;
+use App\Http\Controllers\Controller;
+
+class UserController extends Controller
+{
+    /**
+    * User
+    *
+    * @var User
+    */
+    private $user;
+
+    /**
+    * Construct a UserController
+    *
+    * @param int $user user
+    */
+    public function __construct(User $user)
+    {
+        $this->user = $user;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $users = User::paginate(10);
+        return view('backend.user.list', compact('users'));
```

Comment:
the view file name should match with the controller method name
 please change to backend.user.index